transactions/README.md

```diff
@@ -0,0 +1,59 @@
+# Transactions
+
+## Transaction object
+
+```
+{   
+    typeOfCurrency: string,
+    typeOfTransaction: string,
+    userID: string,
+    amount: number,
+    dateTime: unix-timestamp
+}
+```
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|      [GET /transactions/fetch/:username](#fetch)      | Returns N Transactions in order specified for username provided|
+----
+## **GET /transactions/fetch/:username**
```

Comment:
What's the point of having `fetch` in API endpoint? 🤔 

It could be simply `/transactions/:username` or `/transactions/user/:username` -- needs to be discussed with the FO (Feature Owner)

Reply:
/transactions?limit=10
This should be the endpoint used to fetch the latest transactions.

Reply:
changed it to `/transactions/:username` followed by query params